elastic/kibana
# Combine rule tags with source tags in alert context and AAD for infra rules

Part of https://github.com/elastic/actionable-observability/issues/20

Combines rule tags with source tags in alert's context for Inventory threshold and Metric threshold rules. The same combination of tags is also indexed to AAD.

### Manual Testing
1. Create Inventory rule with some tags
2. Add connector for active and recovered alerts with `{{context}}` in action template
3. Ensure that both rule tags and tags from source are available in `context.tags` in alert notifications
4. Ensure that the combination of tags is also indexed to AAD

Repeat the same steps for Metric threshold rule.
